boards/arm/nrf5340_audio_dk_nrf5340/nrf5340_audio_dk_nrf5340_cpuapp_defconfig

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) 2020 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SOC_SERIES_NRF53X=y
+CONFIG_SOC_NRF5340_CPUAPP_QKAA=y
+CONFIG_BOARD_NRF5340_AUDIO_DK_NRF5340_CPUAPP=y
+
+CONFIG_BOARD_ENABLE_DCDC_APP=y
+CONFIG_BOARD_ENABLE_DCDC_NET=y
+CONFIG_BOARD_ENABLE_CPUNET=y
+
+CONFIG_ARM_MPU=y
+
+CONFIG_HW_STACK_PROTECTION=y
+
+CONFIG_ARM_TRUSTZONE_M=y
+
+CONFIG_PINCTRL=y
+CONFIG_GPIO=y
+CONFIG_NFCT_PINS_AS_GPIOS=y
+
+CONFIG_BT=y
```

Comment:
The basic default for a board needs to have all peripherals disabled and only allows for UART and timer to be enabled